src/pspm_cfg/pspm_cfg_glm_hp_rew.m

```diff
@@ -0,0 +1,50 @@
+function [glm_hp_rew] = pspm_cfg_glm_hp_rew
+% GLM HP FC
+
+% Initialise
+global settings
+if isempty(settings), pspm_init; end;
+
+% set variables
+
+vars = struct();
+vars.modality = 'HPR';
+vars.modspec = 'hp_fc';
+vars.glmref = { ...
+        ['Xia Y, Liu H, Kälin OK, Gerster S, Bach DR (under review). Measuring', ...
+         'human Pavlovian appetitive conditioning and memory retention.']};
+vars.glmhelp = '';
+
+% load default settings
+glm_hp_rew = pspm_cfg_glm(vars);
+
+% set correct name
+glm_hp_rew.name = 'GLM for HP (reward-conditioning)';
+glm_hp_rew.tag = 'glm_hp_rew';
+
+% set callback function
+glm_hp_rew.prog = @pspm_cfg_run_glm_hp_rew;
```

Comment:
There are errors that appear in `pspm_cfg_data_design_selector`, but they have been there before implementing #728 . I will mark the errors in that pull request #728.
<img width="1265" alt="image" src="https://github.com/bachlab/PsPM/assets/3895146/8a4b1b98-e2e9-4a1f-ad1b-97428caa7447">